### PR TITLE
Refactor number extraction from string

### PR DESCRIPTION
**Pull request by dtslvr**

### CODE REVIEW

File: apps/api/src/services/data-provider/ghostfolio-scraper-api/ghostfolio-scraper-api.service.ts [hunk 1]
```diff
@@ -15,6 +15,8 @@ import { PrismaService } from '../../prisma.service';
 
 @Injectable()
 export class GhostfolioScraperApiService implements DataProviderInterface {
+  private static NUMERIC_REGEXP = /[-]{0,1}[\d]*[.]{0,1}[\d]+/g;
```
Comment (gizmodus) [inline apps/api/src/services/data-provider/ghostfolio-scraper-api/ghostfolio-scraper-api.service.ts:18]: A bit easier and also accepting commas:
`private static NUMERIC_REGEXP = /-?\d*(\.|\,)?\d+/g;`

File: apps/api/src/services/data-provider/ghostfolio-scraper-api/ghostfolio-scraper-api.service.ts [hunk 2]
```diff
@@ -75,12 +77,9 @@ export class GhostfolioScraperApiService implements DataProviderInterface {
       const html = await get();
       const $ = cheerio.load(html);
 
-      const string = $(scraperConfig?.selector)
-        .text()
-        .replace('CHF', '')
```
Comment (gizmodus) [inline apps/api/src/services/data-provider/ghostfolio-scraper-api/ghostfolio-scraper-api.service.ts:80]: Is this still needed with the regexp extraction?

**Review by gizmodus — APPROVED** [review]

File: apps/api/src/services/data-provider/ghostfolio-scraper-api/ghostfolio-scraper-api.service.ts (hunk 1 above)
Comment (dtslvr) [reply apps/api/src/services/data-provider/ghostfolio-scraper-api/ghostfolio-scraper-api.service.ts:18]: Thanks, I will add support for commas 👍🏻

File: apps/api/src/services/data-provider/ghostfolio-scraper-api/ghostfolio-scraper-api.service.ts (hunk 2 above)
Comment (dtslvr) [reply apps/api/src/services/data-provider/ghostfolio-scraper-api/ghostfolio-scraper-api.service.ts:80]: It's red -> deleted code 😄

**Pull request merged**